ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -138,6 +139,17 @@ pub struct User {
     pub created_at: DateTime<Utc>,
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct JobRunResult {
```

Comment:
```suggestion
pub struct JobRun {
```

Reply:
Consider move this to job_runs.rs.

In future we'll still need to move this to `schema/mod.rs` (when we add it as part of graphql query), but it provides a better git history by putting to job_runs.rs for now.